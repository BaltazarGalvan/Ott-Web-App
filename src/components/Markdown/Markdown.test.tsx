import React from 'react';
import { render } from '@testing-library/react';

import Markdown from './Markdown';

describe('<Markdownm>', () => {
  test('renders and matches snapshot', () => {
    const markdownString = 'This is a test markdown string with a [testlink](https://www.example.com)';
    const { container } = render(<Markdown markdownString={markdownString} />);

    expect(container).toMatchSnapshot();
  });
  test('marks a link', () => {
    const markdownString = 'This is a test markdown string with a [testlink](https://www.example.com)';
    const { container } = render(<Markdown markdownString={markdownString} />);

    expect(container).toContainHTML('<a href');
  });
});
