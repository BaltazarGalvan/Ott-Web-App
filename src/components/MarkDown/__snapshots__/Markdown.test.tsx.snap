// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Markdownm> renders and matches snapshot 1`] = `
<div>
  <div>
    This is a test markdown string with a 
    <a
      href="https://www.example.com"
      target="_blank"
    >
      testlink
    </a>
  </div>
</div>
`;
